### ISSUE TITLE
Creating element without label

### ISSUE THREAD

**Issue by garethferguson** [dwightwatson/bootstrap-form#54]
Is it possible to create a text element without a label? I've tried setting the value to null and '' but the label is still created.

`BootForm::text('amount', null, '14.50);`

**Comment by dwightwatson** [comment]
Yeah - the package tries to be smart and sort out a label for you. Not a bad idea to have an option to create an input without an input. Happy to take a look at any PRs that implement this.

**Comment by georgearnall** [comment]
I would like to look into this, I've setup the repo how are we supposed to test it? 
I don't quite understand the workflow for developing laravel packages.

**Comment by dwightwatson** [comment]
`composer install` on the repo and you'll be able to use PHPUnit to run the test suite. If you want to test inside a Laravel app you can [define the location of the package using a local URL](http://stackoverflow.com/questions/29994088/composer-require-local-package) so you can test with your clone.

**Comment by georgearnall** [comment]
I got that far, no tests for fields (only open/close forms) ?

**Comment by dwightwatson** [comment]
Looks like it, the test suite can do with some work, I never got around to implementing it completely.

**Comment by georgearnall** [comment]
Ok, well I've sent a pull request with no label options for you to take a look at.
